src/styles/blocks/card.css

```diff
@@ -0,0 +1,90 @@
+
+/* #region card design */
```

Comment:
No needs to use regions in a separate files + there shouldn't be empty lines at the start of your file

Reply:
done